xds/src/test/java/io/grpc/xds/internal/sds/CommonTlsContextTestsUtil.java

```diff
@@ -48,18 +53,83 @@
   public static final String BAD_CLIENT_PEM_FILE = "badclient.pem";
   public static final String BAD_CLIENT_KEY_FILE = "badclient.key";
 
-  static SdsSecretConfig buildSdsSecretConfig(String name, String targetUri, String channelType) {
-    SdsSecretConfig sdsSecretConfig = null;
+  static io.envoyproxy.envoy.api.v2.auth.SdsSecretConfig buildSdsSecretConfigV2(
+      String name, String targetUri, String channelType) {
+    io.envoyproxy.envoy.api.v2.auth.SdsSecretConfig sdsSecretConfig = null;
     if (!Strings.isNullOrEmpty(name) && !Strings.isNullOrEmpty(targetUri)) {
       sdsSecretConfig =
-          SdsSecretConfig.newBuilder()
+          io.envoyproxy.envoy.api.v2.auth.SdsSecretConfig.newBuilder()
               .setName(name)
-              .setSdsConfig(SdsClientTest.buildConfigSource(targetUri, channelType))
+              .setSdsConfig(buildConfigSourceV2(targetUri, channelType))
               .build();
     }
     return sdsSecretConfig;
   }
 
+  private static io.envoyproxy.envoy.extensions.transport_sockets.tls.v3.SdsSecretConfig
```

Comment:
With the import this type doesn't need to be fully qualified, correct?

Reply:
Right. Done.